pyproject.toml

```diff
@@ -17,7 +18,7 @@ azure-identity = "^1.10.0"
 requests-mock = "*"
 pytest = "*"
 azure-core = "*"
-poetry = "*"
+poetry = "^1.1.9"
```

Comment:
Maybe use same as in ae? "^1.7.1"

Reply:
I put 1.1.0, as this was suggested in the error message Morten sent. I think it will default to the newest one.   
Pin poetry@0.6.1 to poetry@1.1.9 to fix
  ✗ Improper Input Validation [Medium Severity][https://security.snyk.io/vuln/SNYK-PYTHON-POETRY-2431202] in poetry@0.6.1